templates/Containerfile

```diff
@@ -47,6 +47,7 @@ COPY --from=ghcr.io/blue-build/cli:
 latest-installer
 {%- endif %} /out/bluebuild /usr/bin/bluebuild
 
+ARG BLUE_BUILD_DATE={{ self::get_build_date() }}
```

Comment:
How about we turn this into a `LABEL` instead and use the constant like we do for `BUILD_ID_LABEL` on line 21.

Reply:
think we can just use BUILD_ID_LABEL